File: base/src/main/java/de/whitefrog/frogr/exception/PersistException.java
package de.whitefrog.frogr.exception;

/**
 * Thrown when something went wrong during persistence.
 * Should be overwritten.
 */
public class PersistException extends FrogrException {
  public PersistException(String message) {
      super(message);
  }
  public PersistException(Throwable cause) {
      super(cause);
  }
  public PersistException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: core/src/main/java/de/whitefrog/frogr/exception/PersistException.kt
package de.whitefrog.frogr.exception

/**
 * Thrown when something went wrong during persistence.
 * Should be overwritten.
 */
open class PersistException : FrogrException {
  constructor(message: String?) : super(message)
  constructor(cause: Throwable) : super(cause)
  constructor(message: String?, cause: Throwable) : super(message, cause)
}
